node/src/chain_spec.rs

```diff
@@ -371,27 +371,27 @@ fn localnet_genesis(
     let mut balances = vec![
         (
             get_account_id_from_seed::<sr25519::Public>("Alice"),
-            1000000000000u128,
+            1000000000000000u128,
```

Comment:
Increase test account genesis balances so they have enough for particular testing operations